_includes/footer.html

```diff
@@ -14,7 +14,7 @@
           </div>
           <div class="hidden-sm hidden-xs col-md-2 innovation-footer-image-wrapper">
              <a href="http://yourfuture.asu.edu/rankings" target="_blank" id="best-colleges-us-news-bage-icon">
-              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme{{site.base_path}}/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
+              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
```

Comment:
I think I've implemented all these changes.

On Wed, Mar 16, 2016 at 9:28 AM, Ryan Raub <notifications@github.com> wrote:

> In _includes/footer.html
> <https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6#discussion_r56366067>
> :
>
> > @@ -14,7 +14,7 @@
> >            </div>
> >            <div class="hidden-sm hidden-xs col-md-2 innovation-footer-image-wrapper">
> >               <a href="http://yourfuture.asu.edu/rankings" target="_blank" id="best-colleges-us-news-bage-icon">
> > -              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme{{site.base_path}}/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
> > +              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
>
> We should have this footer image come from this theme rather than the
> sustainability site.
>
> {{site.base_path}}/assets/bootstrap-asu/build/img/footer/best-colleges-us-news-badge.png
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6/files/47d52f043647bf16b9cdaee60350faf558d55aa2..871a66dda013a725a98e2f7cc75fabc1b75eaaf7#r56366067>
>